### ISSUE TITLE
[Bug]: Selected tab is not very visible under high contrast and dessert mode

### ISSUE THREAD

**Issue by phyyuel** [microsoft/fluentui#33516]
### Component

TabList

### Package version

9.6.2

### React version

17.0.1

### Environment

```shell
Edge chromium latest version
```

### Current Behavior

When using subtle-circular/filled-circular appearance in high contrast aquatic or dessert theme, it is hard to differentiate what tab is selected in the tablist.
![Image](https://github.com/user-attachments/assets/c2cf8e10-ca52-482e-98b6-49c9b41d2fa5)

This is mostly due to having no background color for selected tab in these theme.

### Expected Behavior

The selected tab should be visually obvious in high contrast or dessert mode. 

### Reproduction

https://react.fluentui.dev/?path=/docs/components-tablist--docs#appearance

### Steps to reproduce

1. Navigate to the demo page: https://react.fluentui.dev/?path=/docs/components-searchbox--docs, select a tab in the demo
2. Switch computer theme via windows->settings->Accesscibility->Contrast theme. Select Contrast theme to either Aquatic or Dessert
3. View the demo page and see the selected tab

### Are you reporting an Accessibility issue?

Yes

### Suggested severity

High - No workaround

### Are you willing to submit a PR to fix?

no

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.

**Comment by sopranopillow** [comment]
Makes sense, I think adding the background to match the regular look would help ensure it's understandable. I checked the spec for the circular and it seems like the expectations for HCM were not added, we should double check with a designer before doing this.

**Comment by phyyuel** [comment]
@sopranopillow Is there a way to track the progress? This is critical to our product's A11y review

**Comment by sopranopillow** [comment]
I've assigned it to the team in charge of v9 (cxe-prg) who will triage accordingly, all updates will happen here. If you would like, you could file this issue in here as well: https://dev.azure.com/microsoftdesign/fluent-ui/_wiki/wikis/fluent-ui.wiki/193/Partner-A11y-Issue-Filing-Process

**Comment by phyyuel** [comment]
Thanks. Issue fired: https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/24802
